client/landing/gutenboarding/components/domain-picker/list.tsx

```diff
@@ -37,16 +49,28 @@ const DomainPicker: FunctionComponent< Props > = ( {
 					/>
 				</PanelRow>
 
+				<hr />
```

Comment:
From what I know, `<hr />` is not displayed consistently across browsers so we may want to use something custom here in the future if the design doesn't changes.
I see there are a couple of `Divider` components in calypso but not reused and then there is this [HorizontalRule](https://developer.wordpress.org/block-editor/components/horizontal-rule/) in Gutenberg which for web is just a plain `hr`.

Reply:
Nice! Let's swap for `HorizontalRule` 👍